src/functionalTest/resources/db/testmigration/V1_26__alter_tables_add_columns.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE COURT_VENUE RENAME COLUMN SHORT_COURT_NAME TO EXTERNAL_SHORT_NAME;
```

Comment:
add data to Extertal_Short_Name and verify data loaded for this column for all test data csv files